src/backend/data/feed.js

```diff
@@ -44,6 +51,21 @@ class Feed {
     return isInvalid(this.id);
   }
 
+  /**
+   * Flags a feed in the database, indicating that its processing should be delayed
+   * @param {Number} seconds - duration in seconds for which processing should wait
+   */
```

Comment:
Add another line to this comment:

```js
* Returns a Promise
```

Reply:
Sure, fixed in https://github.com/Seneca-CDOT/telescope/pull/726/commits/850b45fd82e7ffcf6bcc1c91ecc3b4b9e7efd949